File: skill-markets/agent-dev-control-kit/scaffolds/go/files/internal/utils/greet/greet.go
// Package greet provides a small greeting utility.
package greet

import "fmt"

// Greet returns a greeting for the given name.
//
// It returns an error if name is empty.
func Greet(name string) (string, error) {
	if name == "" {
		return "", fmt.Errorf("name must be non-empty")
	}
	return fmt.Sprintf("Hello, %s!", name), nil
}

File: skill-markets/agent-dev-control-kit/scaffolds/python/files/src/my_python_app/utils/greet.py
"""Greeting utility."""


def greet(name: str) -> str:
    """Return a greeting for the given name.

    Args:
        name: Non-empty string.

    Returns:
        Greeting string.

    Raises:
        TypeError: If name is not a string.
        ValueError: If name is empty.
    """
    if not isinstance(name, str):
        raise TypeError("name must be a string")
    if not name:
        raise ValueError("name must be non-empty")
    return f"Hello, {name}!"